src/doc/unstable-book/src/SUMMARY.md

```diff
@@ -106,6 +106,7 @@
 - [link_llvm_intrinsics](link-llvm-intrinsics.md)
 - [linkage](linkage.md)
 - [linked_list_extras](linked-list-extras.md)
+- [linker-flavor](linker-flavor.md)
```

Comment:
> `[linker-flavor]`

It expects this to be snake case I think. A little arbitrary, but it's how it works right now